apps/website/src/app/components/NameGuardWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+
+import { SearchModal, SettingsModal } from "@namehash/nameguard-react";
+
+export function NameGuardWrapper() {
```

Comment:
Right now the NameGuard React pacakge relies on a headlessui float dependency that has to be `use client` so I created this. Importing it as dynamic with ssr disabled didn't resolve it.

I hope to remove the dependency for the Tooltip in another PR as well so we can reduce the number of dependencies needed.